internal/cli/apps.go

```diff
@@ -656,6 +659,69 @@ auth0 apps update <id> -n myapp --type [native|spa|regular|m2m]`,
 	return cmd
 }
 
+func openAppCmd(cli *cli) *cobra.Command {
+	var inputs struct {
+		ID string
+	}
+
+	cmd := &cobra.Command{
+		Use:     "open",
+		Args:    cobra.MaximumNArgs(1),
+		Short:   "Open application settings page in Auth0 Manage",
+		Long:    "Open application settings page in Auth0 Manage.",
+		Example: `auth0 apps open <id>`,
+		PreRun: func(cmd *cobra.Command, args []string) {
+			prepareInteractivity(cmd)
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) == 0 {
+				err := appID.Pick(cmd, &inputs.ID, cli.appPickerOptions)
+				if err != nil {
+					return err
+				}
+			} else {
+				inputs.ID = args[0]
+			}
+
+			cfg := cli.config
+			manageUrl := formatManageURL(cfg, inputs.ID)
+			if manageUrl == "" {
+				cli.renderer.Warnf("Unable to formate the correct URL, please ensure you have run auth0 login and try again.")
+				return nil
+			}
+			err := open.URL(manageUrl)
+			if err != nil {
+				cli.renderer.Warnf("Couldn't open the URL, please do it manually: %s.", manageUrl)
+			}
+			return nil
+		},
+	}
+
+	return cmd
+}
+
+func formatManageURL(cfg config, id string) string {
```

Comment:
Not very familiar with manage, is there a better way to get this?

Reply:
cc @cyx

Reply:
Also, could you please move this function (and `manageDomain`) to the `open` package? https://github.com/auth0/auth0-cli/blob/main/internal/open/open.go